organization/_posts/2018-02-22-startup.md

```diff
@@ -0,0 +1,247 @@
+            
+CWP
+===
+-   ### General Matters and Roadmap
+    -   Michel will gave a presentation about lessons from the CWP and
+        future work at the [4th CERN Scientific Computing
+        Forum](https://indico.cern.ch/event/702775/), on
+        Friday 23 (tomorrow). [Draft
+        slides](https://docs.google.com/presentation/d/1AOkHQ70MGZHXa1PTq_dn1K1qGxjLmQZgfWG75wYJnqo/edit?usp=sharing)
+        available for comments.
+    -   Updated author list version (v3) is live on
+        [arXiv](https://arxiv.org/abs/1712.06982).
+    -   Discussions with the Editors of CSBS advanced - they cannot
+        change the author list after submission (Springer policy). So
+        a 'pre-review' is happening, based on arXiv v3. We can keep
+        the author list open until the Naples meeting.
+    -   Noticed that the CWP inputs are now hidden behind many clicks,
+        so we should make sure they are easier to find. Graeme TODO.
+-   ### Publication strategy for Individual WG Papers
+    -   ### Simulation
+        -   Ready to go to arXiv now, with a finalised author list.
+    -   ### Software Trigger and Event Reconstruction
+        -   Planning to upload both full doc and executive summary
+            tomorrow. Current versions are
+            [https://github.com/davidlange6/recoCWP/blob/master/recoCWP.pdf](https://github.com/davidlange6/recoCWP/blob/master/recoCWP.pdf)
+            and
+            [https://github.com/davidlange6/recoCWP/blob/master/recoCWPFull.pdf](https://github.com/davidlange6/recoCWP/blob/master/recoCWPFull.pdf)
+            so please make any further requests today.
+        -   We asked all the authors to confirm their entry in our
+            author list and got quite a bit of feedback. We've fed
+            this back to the main CWP doc github where relevant.
+    -   ### Data Analysis and Interpretation
+        -   Current version (forked from in HEP-SF/documents, syn'd with
+            origin repo)\
+            [https://github.com/msneubauer/documents/blob/master/CWP/papers/HSF-CWP-2017-05\_analysis/](https://github.com/msneubauer/documents/blob/master/CWP/papers/HSF-CWP-2017-05_analysis/)
+        -   Progress on text this past week, plan to have version
+            uploaded sometime next week
+        
+PyHEP Workshop
+==============
+-   No major update (just 1 extra talk+speaker confirmed since last
+    week).
+    
+Activity updates
+================
+- Eduardo suggested that we should get Zenodo DOIs for the HSF technical
+  notes and also highlight a bit more the TNs (which are a bit hidden).
+  Feedback for this? Also worth for the CWP and CWP chapters?
+  - Generally sounds useful for the DOIs. The website could generally do
+    with some reorganisation to reflect the more active areas and 
+    highlight them more.
+
+Licensing
+---------
+-   HepMCv2 is
+    [relicenced](https://gitlab.cern.ch/hepmc/HepMC/commit/c64061aa2245cd5605306ff292d2ef7a679f063c)
```

Comment:
re-licensed?

Reply:
OED tells me that "relicense" is fine - first use recorded in The Times in 1812 :-)

Reply:
One learns every day :-). Thanks.

Reply:
I do enjoy having the OED at CERN: http://www.oed.com/view/Entry/271157?redirectedFrom=relicense#eid

Reply:
Oh, now I spot it again: my point was about licenSed and not licenCed.

Reply:
Yes, both spellings are fine in English, though people get very excited about them. Again, our friends at the OED say:

> The spelling license, though still often met with, has no justification in the case of the noun. In the case of the verb, on the other hand, although the spelling licence is etymologically unobjectionable, license is supported by the analogy of the rule universally adopted in the similar pairs of related words, practice noun, practise verb, prophecy noun, prophesy verb. (The rule seems to have arisen from imitation of the spelling of pairs like advice noun, advise verb, which expresses a phonetic distinction of historical origin.) A slight argument for preferring the s form in the verb may be found in the existence of the derivatives licensable and licensure (U.S.) which could not conveniently be spelt otherwise.

So the "s" is preferred for the verb, "c" for the noun.

Reply:
Understood. Trying to learn something. Thanks.